conans/client/conan_api.py

```diff
@@ -601,8 +601,6 @@ def install(self, path="", name=None, version=None, user=None, channel=None,
                                         lockfile=lockfile)
 
             install_folder = _make_abs_path(install_folder, cwd)
-            if output_folder:
```

Comment:
Maybe this ``make_abs`` was still correct, when it is defined?

Reply:
Yes. I think this was probably a bug.